spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -264,304 +264,7 @@ If a method of a repository interface has more than one such annotation, the ann
-|Key-value, Wide-Column
-
-|GreaterThan
-|Find results where the property is greater than the given value
-|findByAgeGreaterThan
-|Key-value, Wide-Column
-
-|LessThanEqual
-|Find results where the property is less than or equal to the given value
-|findByAgeLessThanEqual
-|Key-value, Wide-Column
-
-|GreaterThanEqual
-|Find results where the property is greater than or equal to the given value
-|findByAgeGreaterThanEqual
-|Key-value, Wide-Column
-
-|Like
-|Matches String values against the given pattern.
-|findByTitleLike
-|Key-value, Wide-Column, Document, Graph
-
-|IgnoreCase
-|Requests that string values be compared independent of case for query conditions and ordering.
-|findByStreetNameIgnoreCaseLike
-|Key-value, Wide-Column, Document, Graph
-
-|In
-|Find results where the property is one of the values that are contained within the given list
-|findByIdIn
-|Key-value, Wide-Column, Document, Graph
-
-|Null
-|Finds results where the property has a null value.
-|findByYearRetiredNull
-|Key-value, Wide-Column, Document, Graph
-
-|StartsWith
-|Matches String values with the given beginning, which can be a pattern.
-|findByFirstNameStartsWith
-|Key-value, Wide-Column, Document, Graph
-
-|True
-|Finds results where the property has a boolean value of true.
-|findBySalariedTrue
-|Key-value, Wide-Column
-
-|False
-|Finds results where the property has a boolean value of false.
-|findByCompletedFalse
-|Key-value, Wide-Column
-
-|OrderBy
-|Specify a static sorting order followed by one or more ordered pairings of a property path and direction (`Asc` or `Desc`). The direction `Asc` can be omitted from the final property listed, in which case ascending order is implied for that property.
-|findByAgeOrderByHeightDescIdAsc findByAgeOrderById
-|Key-value, Wide-Column
-
-|Desc
-|Specify a static sorting order of descending.
-|findByNameOrderByAgeDesc
-|Key-value, Wide-Column
-
-|Asc
-|Specify a static sorting order of ascending.
-|findByNameOrderByAgeAsc
-|Key-value, Wide-Column
-
-|===
-
-[NOTE]
-====
-The "Not Required For" column indicates the database types for which the respective keyword is not required or applicable.
-====
-
-===== Patterns
-
-Wildcard characters for patterns are determined by the data access provider. For relational databases, `_` matches any one character and `%` matches 0 or more characters.
-
-===== Logical Operator Precedence
-
-For relational databases, the logical operator `And` takes precedence over `Or`, meaning that `And` is evaluated on conditions before `Or` when both are specified on the same method. For other database types, the precedence is limited to the capabilities of the database. For example, some graph databases are limited to precedence in traversal order.
-
-==== Return Types
-
-Refer to the Jakarta Data module Javadoc section on "Return Types for Repository Methods" for a listing of valid return types for methods using Query by Method Name.
-
-=== Special Parameters for Limits, Sorting, and Pagination
```

Comment:
This section applies to various patterns in Jakarta Data and is not unique to Query by Method Name.  It should remain in the main document instead of being moved.

Reply:
To be clear, I'm specifically referring to the section starting from 
```
=== Special Parameters for Limits, Sorting, and Pagination
```
Github also shows some prior lines on my comment, but I am not referring to those preceding lines.

Reply:
@njr-11 This section is on the repository

![Screenshot 2024-04-03 at 17 44 34](https://github.com/jakartaee/data/assets/863011/dc096b6a-a1a2-4464-8c27-b17eca8187a6)